src/server/src/main/java/io/cassandrareaper/storage/cassandra/migrations/Migration025.java

```diff
@@ -45,10 +45,10 @@ public static void migrate(Session session, String keyspace) {
             "INSERT INTO " + V2_TABLE + "(cluster_name, id, repair_run_state) values (?, ?, ?)");
         LOG.info("Converting {} table...", V1_TABLE);
         ResultSet results = session.execute("SELECT * FROM " + V1_TABLE);
-        for (Row row:results) {
+        for (Row row : results) {
```

Comment:
**suggestion:** the style of for loops is inconsistent throughout the codebase. It would be nice to harmonize it and add a checkstyle rule for it. Not in the scope of this PR though.

Reply:
I'm not going to get into this one this time. Seems like a big job.